tasks/main.yml

```diff
@@ -9,4 +9,11 @@
     msg: "The number of hosts in inventory MUST NOT be even"
   when: groups[cassandra_hosts_group] | length is even
 
-- include: install.yml
\ No newline at end of file
+- include: install.yml
+
+- meta: flush_handlers
+
+- include: auth.yml
```

Comment:
auth should be splitted into two parts. One, mandatory, for creating superuser and the second one, optional, for creating other users.